nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss

```diff
@@ -54,7 +50,7 @@
 // To override the canvas theme, you need to set the variables $nifi-theme-light and $nifi-theme-dark
 @import 'assets/themes/nifi';
 // To override the NiFi theme, you need to set the variables $material-theme-light and $material-theme-dark
-@import 'assets/themes/material';
+@import 'assets/themes/purple';
```

Comment:
I'm sure you meant to leave this as the Material theme by default

Reply:
Already fixed my dude!